scripts/repo-add/src/main.rs

```diff
@@ -696,7 +694,7 @@ CREATE TABLE IF NOT EXISTS packages (
     for repo in repos {
         let dbfile_path = format!("{}/{}/pacman.db", *G_TMPWORKINGDIR.lock().unwrap(), repo);
 
-        if cmd_line.ends_with("repo-add") {
+        if cmd_line. == "repo-add" {
```

Comment:
```suggestion
        if cmd_line == "repo-add" {
```

Reply:
Should've tried to compile it before pushing... This time I tested it compiles and functions as expected.